devnotes/mvpd_generation.md

```diff
@@ -0,0 +1,20 @@
+# MVPD data modification
+
+## Testing environement
+
+Original `PNOR` image was aquired by building `op-build`.
+
+`MVPD` partition was read using `pflash` tool:
+```
+pflash -P MVPD -r <output file>
+```
+## Test results
+
+* After flashing original pnor image, `MVPD` partition
+  is empty containg only ECC.
+* After running `hostboot` on default `PNOR` image, `MVPD` partition
```

Comment:
Maybe it makes sense to dump it and link here?

Reply:
Added links to dumps and screenshots